CHANGELOG-add-publication-link 2.md

```diff
@@ -0,0 +1 @@
+- Add link to publications landing page in 'Other' dropdown menu.
```

Comment:
There are duplicated changelog entries because your branch was created from main before the release was made. These will need to be removed if merging main into your branch doesn't do that. Ultimately only your changelog entry should remain.

Reply:
resolved during our meeting